src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Hierarchy.cs

```diff
@@ -86,12 +86,18 @@ internal static async Task<bool> IsOverrideAsync(Solution solution, ISymbol memb
             return await FindImplementedInterfaceMembersArrayAsync(symbol, solution, projects, cancellationToken).ConfigureAwait(false);
         }
 
+        internal static Task<ImmutableArray<ISymbol>> FindImplementedInterfaceMembersArrayAsync(
+            ISymbol symbol, Solution solution, CancellationToken cancellationToken)
+        {
+            return FindImplementedInterfaceMembersArrayAsync(symbol, solution, projects: null, cancellationToken);
```

Comment:
Why this change? I assume we always want to pass a cancellation token?

Reply:
this does pass the cancellation token :)

Reply:
right, the previous overload had it as optional. I am trying to understand why this overload exists

Reply:
This overload exists so the caller doesn't have to provide the project set, but it requires all callers to provide the cancellation token :-)